app/src/main/java/org/jellyfin/mobile/WebappActivity.kt

```diff
@@ -50,12 +55,26 @@ class WebappActivity : AppCompatActivity(), WebViewController {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_webapp)
 
         // Bind player service
         bindService(Intent(this, RemotePlayerService::class.java), serviceConnection, Service.BIND_AUTO_CREATE)
 
+        // Handle window insets
+        setStableLayoutFlags()
+        ViewCompat.setOnApplyWindowInsetsListener(rootView) { v, insets ->
+            v.updatePadding(top = insets.systemWindowInsets.top)
+            insets
+        }
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+            webView.doOnNextLayout { webView ->
+                val verticalCenter = webView.measuredHeight / 2
+                val offset = dip(100)
+                webView.systemGestureExclusionRects = listOf(Rect(0, verticalCenter - offset, dip(96), verticalCenter + offset))
```

Comment:
What do the magic numbers 96 and 100 mean? Why aren't they 50 or 200? Should be more clear IMO

Reply:
The 96 is arbitrary, just picked 2x of the minimum touch target size, probably any other value would also work. The 100 is because I'm offsetting from the center point by 100 in both directions, which results in a height of 200.

Reply:
Please do